### PR TITLE
Fixed spelling mistakes

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -43,9 +43,9 @@ Default controls
 - 0-9: Select item
 - Z: Zoom (needs zoom privilege)
 - T: Chat
-- /: Commad
+- /: Command
 
-- Esc: Pause menu/abort/exit (pauses only singleplayer game)
+- Esc: Pause menu/abort/exit (pauses only single-player game)
```
Comment (SmallJoker) [inline README.txt]: `singleplayer` is used everywhere: [Dev Wiki search results](http://dev.minetest.net/index.php?search=singleplayer&go=Go&title=Special%3ASearch)

**Comment by paramat** [comment]
:+1:

File: README.txt [hunk 2]
```diff
@@ -502,7 +502,7 @@ Lua is licensed under the terms of the MIT license reproduced below.
 This means that Lua is free software and can be used for both academic
 and commercial purposes at absolutely no cost.
 
-For details and rationale, see http://www.lua.org/license.html .
+For details and rationale, see https://www.lua.org/license.html .
```
Comment (DS-Minetest) [inline README.txt]: Actually both links work.

Comment (Bond-009) [inline README.txt]: I know

**Pull request merged**